src/config/dynamic-theme-fixes.config

```diff
@@ -1190,30 +1190,46 @@ img[src*="https://github.githubassets.com/favicon.ico"].smart-link-icon
 img[src^="https://latexmath.bolo-app.com/render/"]
 
 CSS
-#jira-issue-header, #jira-issue-header-actions {
-    background-color: transparent !important;
-}
 span.code {
     background-color: rgba(240,246,252,0.15) !important; 
+    padding: 2px 3px !important;                                                                
 }
-.ghx-parent-group, 
+.ghx-parent-group,
 .ghx-issue {
     background-color: rgba(119, 183, 255, 0.05) !important; 
 }
-span.loader-wrapper a, 
-.css-1ruu48h, 
-.css-tmtnt9, 
-.css-14v1uf5,
+span.loader-wrapper a,
+.css-s0tfqx,
+.css-1vn31bx,
+.css-1gx5gpx,
 .css-wvfva4 {
     background-color: rgba(240,246,252,0.1) !important; 
 }
 .css-1yfnrso,
-.aui-flag, .aui-message {
+.aui-flag, 
+.aui-message {
     box-shadow: 0 0 20px rgba(240,246,252,0.15) !important;
 }
 .css-1ua1xqz {
     background-color: rgba(65, 65, 83, 0.6) !important;
 }
+code:first-of-type {
+    background-image: linear-gradient(to right, rgba(255,255,255,0.03),rgba(255,255,255,0.03)calc(1ch + 16px),transparent calc(1ch + 16px),transparent) !important;
+}
+.ak-editor-panel[data-panel-type="note"] {
+    background-color: rgba(49, 24, 63, 0.71) !important;
+    color: rgba(229, 210, 255, 0.8) !important; 
+}
+.ak-editor-panel[data-panel-type="info"] {
+    background-color: rgba(6, 52, 77, 0.99) !important;
+    color: rgba(200, 230, 255, 0.95) !important;
+}
+.ak-editor-panel[data-panel-type="error"] {
+    background-color: rgba(66, 13, 0, 0.70) !important;
+}
+.ak-editor-panel[data-panel-type="warning"] {
+    background-color: rgba(66, 53, 0, 0.79) !important;
```

Comment:
Please use it like this for every other CSS that's being added:
`${orignialcolor}`
Example:
Originialcolor = red 
then 
```CSS
background-color: ${red}
```
Dark Reader will automatically darken the color for you and take in account the user's settings.

Reply:
Ah alright.. I lowered the vibrancy of red and orange a bit more than how Dark Reader automatically darkened to be able to read a bit better but I guess it's not me that can decide on how bright certain colors should be. 

How do you mean it automatically darkens the color ? Probably I misinterpreted it but I can't get it work for the info panel and note panel. 
* If I do nothing: only warning and error are highlighted https://i.imgur.com/HSauCEG.png 
* If I do like you mentioned its the same as with nothing, although I probably did it wrongly:   https://i.imgur.com/ySfVd9G.png
* With the changes I pushed: https://i.imgur.com/tkOIiPh.png

Naturally, I pressed "Apply" after each change. What am I doing wrong?

Reply:
For the info one you used `()` instead of `{}` simple syntax mistake. And for the note... I'm not sure you could use the originial color in rgb and use that.

Don't forget to add `!important;`

Reply:
It seems to have been the `!important`. I changed the values with the orignial rgb colors and it resulted in the following

![result](https://i.imgur.com/32lBn1m.png)

Reply:
You sure though that its not better to take the "custom" colors I made? How dark reader processes the original colors automatically is very dark and we can barely tell a difference on not so bright screens. Here are the two side to side https://imgur.com/a/JIu3nhl

Reply:
> You sure though that its not better to take the "custom" colors I made?

Eek no. People who tend to have a good amount of customization just take my higher contrast and slightly lower brightness as example I tend to not allow hardcoded values because no matter what they won't change according to the users setting. 

You always can change the color so dark reader will process it differently.

Reply:
> I tend to not allow hardcoded values because no matter what they won't change according to the users setting.

Alright, I understand. 

> You always can change the color so dark reader will process it differently.

I tried to darken the original color but that resulted in a totally wrong generated color. Even though the original was purple, the generated was blueish, so the process is more complicated than just darken the original color. For me right now a big project started, so unfortunately I haven't more time to dedicate fiddling with colors to find a good one. 

What we could do though is take the darkreader formula that darkens the color and reverse my hardcoded colors to get the "bright equivalent" color.

Reply:
> What we could do though is take the darkreader formula that darkens the color and reverse my hardcoded colors to get the "bright equivalent" color.

If you take the the correct color and put that in the `${}`, get the result of that color from the dev-tools and use that as the value for the fix. It should be the exact same color after you put it in `${}`

Reply:
Oh I didn't know this alright, I'll do so.